### ISSUE TITLE
[Mono] Re-enable hardware intrinsics tests when LLVM JIT is enabled for llvmaot lanes

### ISSUE THREAD

**Issue by fanyang-mono** [dotnet/runtime#75767]
The following tests failed on `mono llvmaot Pri0 Runtime Tests Run Linux x64 release` after #75055 disabled LLVM for the JIT fallback. Detailed failure log could be found from [build](https://dev.azure.com/dnceng-public/public/_build/results?buildId=18025&view=logs&j=e8cc9eb8-98f1-55f3-b6df-6c384ccb39b6)

They passed when I tested locally by enabling LLVM JIT. Tests being disabed by #75768 need to be re-enabled.

To enable LLVM JIT, the change to `src/tests/Common/testenvironment.proj`  in #75055 needs to be reverted.

**Comment by jandupej** [comment]
LLVM JIT is no longer supported on Mono side. Closing.

**Comment by fanyang-mono** [comment]
@jandupej It is a good idea to close this issue according to the reason you mentioned above. However, the associated tests are still disabled in `issues.target` file with a link to this issue. It would be helpful to link those tests with a new issue. It looks like that they are all X86 intrinsics related. 

For example:
https://github.com/dotnet/runtime/blob/main/src/tests/issues.targets#L2282-L2365
